### PR TITLE
Make Python package spec indirect

### PR DESCRIPTION
**Pull request by soltanmm**
@nathanielmanistaatgoogle Please review and merge!

### CODE REVIEW

**Comment by nathanielmanistaatgoogle** [comment]
LGTM but I'd like a C++ reviewer to also have a look.

@nicolasnoble, please review and merge?

File: src/compiler/python_generator.cc [hunk 1]
```diff
@@ -53,8 +58,11 @@ using google::protobuf::Descriptor;
 using google::protobuf::FileDescriptor;
 using google::protobuf::MethodDescriptor;
 using google::protobuf::ServiceDescriptor;
+using google::protobuf::compiler::GeneratorContext;
+using google::protobuf::io::CodedOutputStream;
 using google::protobuf::io::Printer;
 using google::protobuf::io::StringOutputStream;
+using google::protobuf::io::ZeroCopyOutputStream;
```
Comment (nicolasnoble) [inline src/compiler/python_generator.cc]: It's fine for now, but we should use the compiler/config.h file to use grpc::protobuf instead. For a future PR that is.

File: build.json [hunk 2]
```diff
@@ -492,6 +492,19 @@
       "secure": "no"
     },
     {
+      "name": "grpc_python_plugin_support",
```
Comment (nicolasnoble) [inline build.json]: Why did you want to split that to a separate library ?

Comment (soltanmm) [inline build.json]: To be able to target that library internally.

Comment (nicolasnoble) [inline build.json]: Fair enough.

**Pull request merged**